client/src/components/Feed/PostDeleteModal.js

```diff
@@ -0,0 +1,59 @@
+import { Modal } from "antd";
+import styled from "styled-components";
+
+// Constants
+import { theme, mq } from "constants/theme";
+
+const { colors, typography } = theme;
+const { royalBlue } = colors;
+const { large, xlarge } = typography.size;
+
+const PostDeleteModal = styled(Modal)`
+    .ant-modal-content {
+        filter: drop-shadow(0px 0px 8px rgba(0, 0, 0, 0.12));
+        border-radius: 10px;
+        .ant-modal-header {
+            border-bottom: 1px solid #C5C5C5;
+            border-radius: 10px 10px 0 0;
+            padding-bottom: 0;
+            .ant-modal-title {
+                text-align: center;
+                font-weight: 700;
+                font-size: ${large};
+            }
```

Comment:
Same idea as above in this file, there is a mix here, rems & pxs, just use `rem`... @Zaydme or anyone else, keep a note to update later on...